src/WebUI/locales/en.yml

```diff
@@ -1019,12 +1019,19 @@ item:
     Cut:
       long: 'Cut'
       short: 'c'
+      title: 'Cut damage'
+      description: 'Cut damage description'
     Pierce:
       long: 'Pierce'
       short: 'p'
+      title: 'Pierce damage'
+      description: 'Pierce damage description'
```

Comment:
Pierce damage, on the other hand, signifies the damage dealt by weapons designed to puncture or penetrate, such as arrows, spears, or daggers. The key strength of pierce damage lies in its efficiency against armoured opponents. Pierce damage offers a headshot multiplier increase greater than usual, if a melee (pierce) attack lands on an enemy's head, the damage increases by 1.3 times, while for ranged attacks, the increment is 1.6 times. This makes pierce damage the strongest potential damage type given its benefit from headshots.